project_task_schedule/models/project_task.py

```diff
@@ -0,0 +1,25 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models, api
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+
+    enable_task_planning = fields.Boolean(
+        related='project_id.enable_task_planning')
+    date_range_id = fields.Many2one(
+        comodel_name="date.range", string="Date range")
+    start_date = fields.Date("Start Date")
+    end_date = fields.Date("End Date")
```

Comment:
If it is two new fields, I like `date_planned_start` and `date_planned_end`, mirroring Enterprise; if we use existing fields, I think `date_end` is still there and we can reimplement `date_start` which was removed in 13.0. Let's see what Pedro replies on my question

Reply:
He answered `date_start` and `date_end`, so let's reimplement date_end and then propose it to him and see what he says